golem-ts-agentic/packages/golem-ts-typegen/tests/testData.ts

```diff
@@ -159,6 +161,7 @@ class MyAgent {
     recursiveType: RecursiveType,
     objectWithTypeParameter: ObjectWithTypeParameter<["en", "de"]>,
     unionWithTypeParameter: UnionWithTypeParameter<["en", "de"]>,
+    multimodal: MultimodalType<string | boolean>,
```

Comment:
This is just a test specific in typegen module that it can handle generic type parameter.